miden-test-utils/src/tester.rs

```diff
@@ -66,19 +67,22 @@ impl Test {
 
     /// Asserts that running the test for the expected TestError variant will result in an error
     /// that contains the TestError's error substring in its error message.
-    pub fn expect_error(&self, error: TestError) {
+    pub fn expect_error<L>(&self, error: TestError, libraries: Vec<L>)
```

Comment:
It is probably better to modify `bult_test!` macro so that it can take a vector of libraries and store it in the `Test` struct rather than passing libraries via `expect_error()`, `epxect_stack()` etc.

Reply:
Absolutely, let me try to get it done.

Reply:
Doesn't seem like this has already been done, right?